manageability-docker/TBH/docker-ble-policy

```diff
@@ -0,0 +1,47 @@
+#include <tunables/global>
+
+profile docker-ble-policy flags=(attach_disconnected,mediate_deleted) {
```

Comment:
change docker-ble-policy -> docker-manageability-policy (ble stands for bluetooth low energy)